css/css-masking/animations/clip-path-interpolation-shape.html

```diff
@@ -0,0 +1,170 @@
+}, [
+  {at: -0.3, expect: 'shape(from calc(-4.5% - 6.5px) 2px, move to 0.5% 4.3px)'},
+  {at: 0, expect: 'shape(from calc(0% - 5px) 5px, move to 5% 1px)'},
+  {at: 0.5, expect: 'shape(from calc(7.5% - 2.5px) 10px, move to 12.5% -4.5px)'},
+  {at: 1, expect: 'shape(from 15% 15px, move to 20% -10px'},
+  {at: 1.5, expect: 'shape(from calc(22.5% + 2.5px) 20px, move to 27.5% -15.5px)'},
+]);
+
+test_no_interpolation({
+  property: 'clip-path',
+  from: 'unset',
+  to: 'shape(from 10px 10px, close)',
+});
+
+test_no_interpolation({
+  property: 'clip-path',
+  from: 'none',
+  to: 'shape(from 10px 10px, close)',
+});
+
+test_no_interpolation({
+  property: 'clip-path',
+  from: 'shape(from 10px 10px, move to 10% 10%)',
+  to: 'shape(from 10px 10px, close)',
+});
+
+
+test_interpolation({
+  property: 'clip-path',
+  from: 'shape(from 5% 5px, hline to 5%, vline to -5px, close)',
+  to: 'shape(from 15% 15px, hline to 25%, vline to -15px, close)',
+}, [
+  {at: -0.3, expect: 'shape(from 2% 2px, hline to -1%, vline to -2px, close)'},
+  {at: 0, expect: 'shape(from 5% 5px, hline to 5%, vline to -5px, close)'},
+  {at: 0.5, expect: 'shape(from 10% 10px, hline to 15% , vline to -10px, close)'},
+  {at: 1, expect: 'shape(from 15% 15px, hline to 25%, vline to -15px, close)'},
+  {at: 1.5, expect: 'shape(from 20% 20px, hline to 35%, vline to -20px, close)'},
+]);
+
+test_interpolation({
+  property: 'clip-path',
+  from: 'shape(from 5% 5px, curve to 10% 10px via 0% 80px, curve to 30% 20px via 20% 50px 25% 70px)',
+  to: 'shape(from 15% 15px, curve to 20% 0px via 10% 60px, curve to 20% 30px via 30% 40px -5% 100px)',
+}, [
+  {at: -0.3, expect: 'shape(from 2% 2px, curve to 7% 13px via -3% 86px, curve to 33% 17px via 17% 53px 34% 61px)'},
+  {at: 0, expect: 'shape(from 5% 5px, curve to 10% 10px via 0% 80px, curve to 30% 20px via 20% 50px 25% 70px)'},
+  {at: 0.5, expect: 'shape(from 10% 10px, curve to 15% 5px via 5% 70px, curve to 25% 25px via 25% 45px 10% 85px)'},
+  {at: 1, expect: 'shape(from 15% 15px, curve to 20% 0px via 10% 60px, curve to 20% 30px via 30% 40px -5% 100px)'},
+  {at: 1.5, expect: 'shape(from 20% 20px, curve to 25% -5px via 15% 50px, curve to 15% 35px via 35% 35px -20% 115px)'},
+]);
+
+test_interpolation({
+  property: 'clip-path',
+  from: 'shape(from 5% 5px, curve by 10% 10px via 0% 80px, curve by 30% 20px via 20% 50px 25% 70px)',
+  to: 'shape(from 15% 15px, curve by 20% 0px via 10% 60px, curve by 20% 30px via 30% 40px -5% 100px)',
+}, [
+  {at: -0.3, expect: 'shape(from 2% 2px, curve by 7% 13px via -3% 86px, curve by 33% 17px via 17% 53px 34% 61px)'},
+  {at: 0, expect: 'shape(from 5% 5px, curve by 10% 10px via 0% 80px, curve by 30% 20px via 20% 50px 25% 70px)'},
+  {at: 0.5, expect: 'shape(from 10% 10px, curve by 15% 5px via 5% 70px, curve by 25% 25px via 25% 45px 10% 85px)'},
+  {at: 1.5, expect: 'shape(from 20% 20px, curve by 25% -5px via 15% 50px, curve by 15% 35px via 35% 35px -20% 115px)'},
+]);
```

Comment:
Is this duplicated? Looks like this is the same as the previous one.

Reply:
It uses `by` instead of `to`